components/HeroUnit.js

```diff
@@ -84,3 +84,7 @@ function HeroUnitFooter () {
     </div>
   )
 }
+
+function ForDownArrowIconHandler () {
```

Comment:
@ratchagarn By JavaScript convention, I think it's better to set the method name starting with small case krub.